bundles/azure/aso/psql/bundle/config/00-schema.yml

```diff
@@ -0,0 +1,72 @@
+#@data/values-schema
+---
+#@schema/title "ASO Controller Namespace"
+#@schema/desc "The Namespace where the Azure ASO controller is installed that should own this Azure RM resource"
+aso_controller_namespace: ""
+
+#@schema/title "Create namespace flag"
+#@schema/desc "Whether to create or not the namespace for the resources"
+#@schema/type any=False
+#@schema/default True
+create_namespace: True
+
+#@schema/title "ResourceGroup"
+#@schema/desc "Azure ResourceGroup for the servers/database resources"
+resource_group: ""
+
+#@schema/title "UseExistingResourceGroup"
+#@schema/desc "Whether to use an existing Azure resource group or not"
+#@schema/default False
+use_existing_resource_group: True
```

Comment:
question: with `#@schema/default False` and `use_existing_resource_group: True`:
- what is actually the default now?
- is there a reason why we set both?

(same questions apply for other data values)

Reply:
The value used determines the Type, in this case Boolean.
The default value sets the default value.
We cannot set a default value to a Boolean value if there is no Type.

Though I guess we can set it to `use_existing_resource_group: False` to reduce the confusion.
I believe we did this to answer the questions you asked but did not cleanup.

Reply:
We cleaned up the superfluous `#@schema/default` statements.